assets/css/main.css

```diff
@@ -51,7 +51,7 @@
 }
 
 body {
-  overflow-y: scroll;
+  /* overflow-y: scroll; */
```

Comment:
Why removing it?

Reply:
If I remember, with this overflow AppHeader is loosing it's position. Some stacking context weirdness.